### PR TITLE
Create similar players list

### PR DESCRIPTION
**Pull request by f117qaz123**

### CODE REVIEW

**Comment by f117qaz123** [comment]
[Create SimilarPlayersList](https://trello.com/c/8t0KGGL5/12-create-similarplayerslist)

File: client/src/components/PlayerPage/PlayerPage.tsx [hunk 1]
```diff
@@ -27,8 +31,14 @@ const PlayerPage = () => {
 
   const playerName = data.players[0].short_name;
 
+  const playerOverall = data.players[0].overall;
+
+  const playerPotential = data.players[0].potential;
```
Comment (LianyuMa) [inline client/src/components/PlayerPage/PlayerPage.tsx]:
```suggestion
  const [{playerName, playerOverall, playerPotential}] = data.players;
```

Comment (f117qaz123) [reply client/src/components/PlayerPage/PlayerPage.tsx:36]: image

Comment (f117qaz123) [reply client/src/components/PlayerPage/PlayerPage.tsx:36]: image

**Review by LianyuMa — APPROVED** [review]

**Pull request merged**